contents/blog/startup-ops-toolkit.md

```diff
@@ -13,63 +13,66 @@ Ok, so you’ve read *Zero to One* and *The Hard Thing About Hard Things*, but w
 
 <iframe src="https://giphy.com/embed/j6wvxJLueHV10lATCF" width="480" height="360" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/usnationalarchives-archivesgif-youinpublicservice-publicservice-j6wvxJLueHV10lATCF">via GIPHY</a></p>
 
-More importantly, what are the tools you can use to streamline and automate these?
+More importantly, what is your ops stack you can use to streamline and automate these?
 
-**Spoiler alert**: it’s Deel + Gusto + Fondo + Pry + Brex + some free contract templates + Google Sheets/Docs + Workable + Hubspot + Charlie
-
-This article is US-focused, but most of it applies to any country. I won’t offer multiple options here - these are the tools that I have settled on using personally after a bunch of research and trying out different things.
+This article assumes you work at a US company, but most of it applies to any country. I won’t offer multiple options here - these are the tools that we have settled on using at PostHog after a bunch of trial and error (and tedious customer demos). 
 
 By using these, hopefully I can save you a bunch of time and precious $$$ trying to figure out how to put together contracts, pay people on time, manage your money and comply with laws you are only just finding out exist.
 
```

Comment:
We should add a author profile for you here outlines who you are and your expertise in the area. Something like on: https://posthog.com/blog/missing-recordings